iana-mediatypes/oci-mediatype.md

```diff
@@ -0,0 +1,66 @@
+**Name:** Steve Lasker
+
+**Email:** tob@opencontainers.org
+
+**Media type name:** application
+
+**Media subtype name:** application/vnd.oci.image.manifest.v1+json
+
+**Required parameters:** n/a
+
+**Optional parameters:** n/a
+
+**Encoding considerations:** binary
+
+**Security Considerations:** This JSON format media type inherits the security considerations for JSON. See RFC 8259 section 12. (https://tools.ietf.org/html/rfc8259#section-12)
+
+**Privacy Information:** The manifest contains configuration information, with optional annotations. While users may put any content they wish into these annotations, there are no requirements for privacy information that must be protected.
+
+**Extensibility:** The json payload supports annotations as strings. No executable information or code paths are enabled.
+
+**Compression:** The manifest type is not compressed in normal workflows.
+
+**Links:** The manifest contains digests of other linked content, which are separately retrieved, with their own security considerations.
+
+**Interoperability Considerations:** An OCI image manifest may represent one of many different platforms and architectures. When retrieving a manifest, the config descriptor https://github.com/opencontainers/image-spec/blob/master/config.md returns a an object the consumer can determine if this manifest applies to the host platform and architecture.
+
+**Published specification:**
+application/vnd.oci.image.manifest.v1+json spec: https://github.com/opencontainers/image-spec/blob/master/manifest.md
+Content Digest spec: https://github.com/opencontainers/distribution-spec/blob/master/spec.md#content-digests
```

Comment:
> IANA: This reference doesn't appear to be correct. Isn't the correct reference for this specific format

 Fixed link

Reply:
the content digest link isn't needed here..